docs/specs.md

```diff
@@ -68,14 +78,30 @@ _An enumeration of possible transformation use cases and the requirements they w
 
 ## Data Specification
 
-The data formats for the Detection block will be:
+Taking the interfaces of the Detection block as a central point, we can define:
+
+* as _input_, a **Source Document**, either generated directly or via the
+  Preprocessing block (not that this source document is the _output_ data
+  specification for the Preprocessing block)
+* as _output_, a **PII collection**, containing a description of PII
+  elements. A subspecification inside it is the **PII Detector**, describing a
```

Comment:
yes, we should aim for consistency, to avoid confusion.

One thing we could do is to consider "Recognition" the *full* process of extracting consolidated PII instances. In that sense, _Recognition_ would have two parts
 * _Detection_ is the first phase, in which matches are done
 * _Decision_ is the second phase that consolidates all detected PII instances

Would that work?

Reply:
Yes it could work. In this case we should rename the block in the diagram to "PII Detector". I can make this change after the PR is merged.